From 3cf507ddec0e4ae3e0f310d0a822414a83e9df35 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Matthias=20Bl=C3=BCmel?= <blaimi@blaimi.de>
Date: Thu, 21 Jan 2021 22:52:05 +0100
Subject: [PATCH libwacom 09/10] data: Add Lenovo ThinkPad X13 Yoga Gen 1
 (ISDv4 5220) (#336)

(cherry picked from commit cd48da82f01e9e60501adc3cc96a668349079cd2)
---
 data/isdv4-5220.tablet | 22 ++++++++++++++++++++++
 1 file changed, 22 insertions(+)
 create mode 100644 data/isdv4-5220.tablet

diff --git a/data/isdv4-5220.tablet b/data/isdv4-5220.tablet
new file mode 100644
index 0000000..c1bba66
--- /dev/null
+++ b/data/isdv4-5220.tablet
@@ -0,0 +1,22 @@
+# Lenovo ThinkPad X13 Yoga Gen 1
+# Sensor Type: AES
+# Features: Touch (Integrated), Tilt
+# HW Resolution: 29376 x 16524 (2540 x 2540 lpi)
+#
+# Autogenerated from sysinfo.KNB1IfMgnC
+# https://github.com/linuxwacom/wacom-hid-descriptors/issues/129
+
+[Device]
+Name=ISDv4 5220
+ModelName=
+DeviceMatch=usb:056a:5220
+Class=ISDV4
+Width=12
+Height=7
+IntegratedIn=Display;System
+Styli=@isdv4-aes;
+
+[Features]
+Stylus=true
+Touch=true
+Buttons=0
-- 
2.31.1

